### PR TITLE
Added docker to documentation

### PR DESCRIPTION
**Pull request by omeraha**
## Description

1.  Added docker usage to README file.

1. Passed `--directory` arg to pie-my-vulns.js to the docker file ENTRYPOINT command.

1. Fix double printing of "Scanning dependencies" title.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue
Closes the following issues:

1. https://github.com/lirantal/pie-my-vulns/issues/35

1. https://github.com/lirantal/pie-my-vulns/issues/51

## Motivation and Context

1. Provide information about docker container usage

1. Allow passing the target directory to the pie-my-vulns command when running it inside the container.

## How Has This Been Tested?

1. Docker related changes were tested manually. Automated test will be added as part of issue https://github.com/lirantal/pie-my-vulns/issues/52

1. Double title print fix testing is already covered by test automation

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM node:13.8.0-alpine3.10
+FROM node:12.16.2-alpine3.9
```
Comment (lirantal) [inline Dockerfile:1]: was there a specific reason we moved to this one?

**Comment by lirantal** [comment]
Looks great @omeraha, thanks!

File: Dockerfile (hunk 1 above)
Comment (omeraha) [reply Dockerfile:1]: Yes. Node.js version 13.x.x will become unsupported soon (as all odd numbers versions). Meanwhile, version 12.x.x is widely used and will be supported until October 2021

**Pull request merged**